build.gradle.kts

```diff
@@ -1,6 +1,8 @@
 plugins {
+    val agpVersion = System.getenv("VERSION_AGP") ?: BuildPluginsVersion.AGP
```

Comment:
this line is duplicated across build scripts, lets create (Dependencies.kt) a method that does the same check and just call it from here, wdyt?

Reply:
Yup let's do it.